aboutme.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About me!</title>
+    <link rel="stylesheet" href="Aboutme.css">
+  </head>
+  <body>
+    <h1>Day 2 assingment About me!</h1>
+    <p><strong> Hello! My name is Adan Rabadan.</strong><br />I'm atteding Code fellows to become a software developer.
+      Before I attended code fellows i was a machinist for aircraft manufactor,<br /> prior to that I served
```

Comment:
you lose your indentation on this line.  lines 10-21 should be indented to one less level